sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -652,6 +653,8 @@ export interface DownloadFromBlobOptions {
   parallelism?: number;
 }
 
+export type GenerateSASUrlOptions = Omit<BlobSASSignatureValues, "blobName" | "containerName">;
```

Comment:
[@jeremymeng's comment from HarshaNalluru/azure-sdk-for-js/pull/21 ]

This is an amazing feature! I wonder what the coding experiences would be though. Would Go to Definition come here or go to BlobSASSignatureValues directly? Some time it is easier to understand what's available than what's not available. Maybe Pick is better? 
/cc @bterlson

Reply:
`Omit` is a feature that was introduced in TypeScript 3.5 which is newer than the lowest version we support (3.1). Instead, we've been using a combination of Pick and Exclude in App Configuration which are both available in the versions of TypeScript we support:

Example
https://github.com/Azure/azure-sdk-for-js/blob/834a68404c56cebe3d125c168aab43db6cb1611c/sdk/appconfiguration/app-configuration/src/models.ts#L83

Reply:
Yes, Ramya mentioned this.